frontend/src/features/admin-form/common/mutations.ts

```diff
@@ -52,8 +53,8 @@ enum FormCollaboratorAction {
 }
 
 export const useMutateCollaborators = () => {
-  const { formId } = useParams()
-  if (!formId) throw new Error('No formId provided')
+  const { formId } = useCollaboratorWizard()
+  if (!formId) throw new Error('No formId provided to useMutateCollaborators')
```

Comment:
no need to throw here if we enforce the presence of formId in the first place. Doing this means possible run-time errors